layout/js/map.js

```diff
@@ -157,10 +158,12 @@ function hovertext(text, evt){
     stateVal = " ";
   } else {
     var ref = evt.target.getAttribute('xlink:href').split('-')[0];
-    var stateName = ref.replace(/#/g, '')
+    var stateName = ref.replace(/#/g, '');
     var displayNum = Math.round(get_state_value(stateName));
     if (isNaN(displayNum)){
       displayNum = 'no data';
+    } else {
+      displayNum = displayNum.toLocaleString() + ' mgd';
```

Comment:
Oh. yeah. That toLocaleString should work on all browsers, but I see some notes that it may not be universal. @jiwalker-usgs should weigh in on this method.